src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -14,6 +15,14 @@ internal static class EnvironmentExtensions
         // For testing
         internal static string BaseDirectory { get; set; }
 
+        public static bool IsOutOfProc(this IEnvironment environment)
+        {
+            // TODO: need to also handle HttpWorker scenario?
```

Comment:
@Pragna - what's the right way to determine whether we're in an httpworker scenario here? There's a few places I'm now using this new IsOutOfProc check to determine whether invocations are going OOP.

Reply:
here is how we check for httpworker: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Host/ScriptHost.cs#L129.